### ISSUE TITLE
Popper won't flip correctly on IE

### ISSUE THREAD

**Issue by claudiobmgrtnr** [FezVrasta/popper.js#65]
### CodePen demo

http://codepen.io/claudiobmgrtnr/pen/grBrmV

### Steps to reproduce the problem

**NOTE: The bug will only appear in IE10 or IE11. Edge works fine.**

1. Open any popper whose boundary element is "viewport"
2. Apply "overflow-y: auto" to the body
3. Scroll until the popper should flip to prevent itself exiting the viewport

### What is the expected behavior?

The popper should flip to the correct position following the order of the desired flip behavior.

### What went wrong?

The popper exits the viewport. There seems to be a problem in IE where scrolltop of the body will always be 0 if its "overflow-y" css property is set to "auto". This causes wrong calculation results of the boundaries. Removing the overflow-y property should fix it. 

### Any other comments?
I didn't test it in IE9

**Comment by FezVrasta** [comment]
I don't see the popper at all in your demo..

**Comment by claudiobmgrtnr** [comment]
there should be a button in between of the text that says "click me" that toggles the popper.

**Comment by FezVrasta** [comment]
Ok I reproduced the problem on IE11... 

Seems like we have to use:

```javascript
document.documentElement && document.documentElement.scrollTop
```

To properly get the `scrollTop` value on IE11 and below.

**Comment by FezVrasta** [comment]
We already do this kind of check because it was needed for Firefox compatibility.. I'm investigating why it doesn't work..

**Comment by FezVrasta** [comment]
Seems like I patched it, here a demo:
http://codepen.io/FezVrasta/pen/XKxLkz

I'm going to create a PR to see if the tests still pass and if so I'll merge the fix

**Comment by FezVrasta** [comment]
Fixed on 0.5.3, released right now
